test/index.js

```diff
@@ -1652,15 +1659,19 @@ suite ('laws', () => {
   });
 
   suite ('RegExp', () => {
-    const arb = jsc.string.smap (s => {
-      const validFlags = ['', 'g', 'i', 'm', 'gi', 'gm', 'im', 'gim'];
-      const flags = validFlags[Math.floor (Math.random () * validFlags.length)];
```

Comment:
While I was at it, I also removed this problematic code. One of the benefits of prop tests is that you can reproduce a failing assertion by rerunning the tests with the same seed. Of course this depends on your arbitraries being defined in terms of pure functions.